packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -696,6 +796,18 @@ class SemanticsProperties extends DiagnosticableTree {
   ///  * [FocusNode], [FocusScope], [FocusManager], which manage the input focus
   final VoidCallback onDidLoseAccessibilityFocus;
 
+  /// A map from each supported [CustomSemanticsAction] to a provided handler.
+  /// 
+  /// The handler associated with each custom action is called whenever a
```

Comment:
Should this doc also be updated to the version you are using above?

Reply:
Do you mean the CustomSemanticsAction doc comment?